### ISSUE TITLE
try to support encoding molecules without a detached encoding dictionary

### ISSUE THREAD

**Issue by UnixJunkie** [UnixJunkie/molenc#111]
This is more convenient for users.
Though harder to program/handle.
Also, there might need dictionary translation
from a test set back to the features of the training set.
I.e. encoded molecules will have the dictionary as a header
in the output file.

**Comment by UnixJunkie** [comment]
will need to be turned on with a special CLI option, because will break compatibility w/ previous versions for many tools

**Comment by UnixJunkie** [comment]
This might be dangerous and loose molecular information: only test set features will be translated
back to training set features.
Maybe log untranslated feature statistics, as a precaution and to detect when things are derailing.

**Comment by UnixJunkie** [comment]
Unknown features should be counted in the first feature index of each encoded molecule (index 0 or 1?).
The file should have all atom types as header (decr. frequency sort).
Followed by all pair types (decr. frequency sort).
Then encoded molecules (same as input order).

**Comment by UnixJunkie** [comment]
create a new molenc_AP exe for all this

**Comment by UnixJunkie** [comment]
not so much a good idea
